.github/workflows/renovate-release-json.yaml

```diff
@@ -0,0 +1,63 @@
+name: Fetch Releases and Generate JSON
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  generate-json:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      
+      - name: "Setup Go Env"
+        uses: "actions/setup-go@v4"
+        with:
+          cache: true
+          go-version: "1.19.2"
+
+      - name: Generate JSON File
+        run: |
+          go run renovate.go
+
+      - name: "Ensure Private Key"
+        env:
+          BOT_SSH_KEY: '${{ secrets.BOT_SSH_KEY }}'
+        run: |
+          echo "$BOT_SSH_KEY" > /tmp/id_ed25519
+
+      - name: "Setup SSH Agent"
+        env:
+          SSH_AUTH_SOCK: "/tmp/ssh_agent.sock"
+        run: |
+          mkdir -p ~/.ssh
+          ssh-keyscan github.com >> ~/.ssh/known_hosts
+          ssh-agent -a ${SSH_AUTH_SOCK} > /dev/null
+          chmod 0600 /tmp/id_ed25519
+          ssh-add /tmp/id_ed25519
+
+      - name: "Setup Git Config"
```

Comment:
I think we have this simpler git config setup for GitHub actions https://github.com/giantswarm/releases/blob/cc714ffd0d84370138f68bc91a3dcb103afbbd34/.github/workflows/create-release.yaml#L51

Reply:
In my opinion it should be committed by the owner, e.g. you merger CAPA Release v27.0.0. Releases.json should be updated from the one who merged it.